src/figvalidationservice/figcrpords.yaml

```diff
@@ -327,5 +334,16 @@ definitions:
         type: string
       status_message:
         type: string
-
- 
\ No newline at end of file
+  validateOrgApplicantServiceOrdsResponse:
```

Comment:
Can we change this to be validateApplicantOrdsResponse instead of Validate**Org**ApplicantOrdsResponse?

Reply:
Class name changed as suggested.

Reply:
Thanks